src/__main__.py

```diff
@@ -66,37 +112,52 @@ def get_code_duplication_by_range_date(
             directory_path,
             get_commit_by_date(directory_path, date, current_branch),
         )
-        code_duplications_percentage_by_date.append(
+        debt_report.append(
             {
-                "DATE": date,
+                "DATE": subtract_day_to_date(date, 1),
                 "CODE_DUPLICATION": get_code_duplication_percentage(
                     directory_path
                 ),
+                "IMPLEMENTATION_LINES": get_implementation_and_test_lines(
+                    directory_path, file_extensions, "implementation_lines"
+                ),
+                "TEST_LINES": get_implementation_and_test_lines(
+                    directory_path, file_extensions, "test_lines"
+                ),
+                "TOTAL_LINES": get_implementation_and_test_lines(
+                    directory_path, file_extensions, "total_lines"
+                ),
             }
         )
     checkout_by_commit_or_branch(directory_path, current_branch)
-    return code_duplications_percentage_by_date
-
-
-def format_debt_report(code_duplication_list):
-    return "Date;Code Duplication\n" + "\n".join(
-        [
-            f"{code_duplication['DATE']};{code_duplication['CODE_DUPLICATION']}"
-            for code_duplication in code_duplication_list
-        ]
+    return debt_report
+
+
+def format_debt_report(debt_list):
+    return (
+        "Date;Code Duplication;Implementation Lines;Test Lines; Total Lines\n"
+        + "\n".join(
+            [
+                f"{debt['DATE']};{debt['CODE_DUPLICATION']};"
+                f"{debt['IMPLEMENTATION_LINES']};"
+                f"{debt['TEST_LINES']};{debt['TOTAL_LINES']}"
+                for debt in debt_list
```

Comment:
Aquí en general, en lugar de `for elt in elt_list`, usamos `for elt in elts`. Por ej. `for dog in dogs`, o `for cat in cats`. Realmente que sea una lista, o un set, o una tupla, o lo que sea, no importa tanto. Lo que importa, es que sea iterable.